packages/dbml-core/src/export/DbmlExporter.js

```diff
@@ -263,59 +266,67 @@ class DbmlExporter {
     return strArr.length ? strArr.join('\n') : '';
   }
 
+  static getTableGroupSettings (tableGroup) {
+    const settings = [];
+
+    if (tableGroup.color) settings.push(`color: ${tableGroup.color}`);
+
+    return settings.length ? ` [${settings.join(', ')}]` : '';
+  }
+
   static exportTableGroups (tableGroupIds, model) {
     const tableGroupStrs = tableGroupIds.map(groupId => {
       const group = model.tableGroups[groupId];
       const groupSchema = model.schemas[group.schemaId];
+      const groupSettingStr = DbmlExporter.getTableGroupSettings(group);
 
-      return `TableGroup ${shouldPrintSchema(groupSchema, model)
-        ? `"${groupSchema.name}".` : ''}"${group.name}" {\n${
-        group.tableIds.map(tableId => {
+      const groupNote = group.note ? `  Note: ${DbmlExporter.escapeNote(group.note)}\n`: '';
+      const groupSchemaName = shouldPrintSchema(groupSchema, model) ? `"${groupSchema.name}".` : '';
+      const groupName = `${groupSchemaName}"${group.name}"`;
+
+      const tableStr = group.tableIds
```

Comment:
`tableList` is a better name here. It avoid making confusion with the `tableStrs` above which store the table definitions.

Reply:
Or it should be: `tableNames` instead since it only a list of table names.

Reply:
I updated it as `tableNames`